ha/setup/ha_setup.py

```diff
@@ -521,6 +521,7 @@ def process(self):
         else:
             # configure stonith for each node from that node only
             if enable_stonith:
+                self._execute.run_cmd(const.PCS_CLEANUP)
```

Comment:
Please add cleanup in all
if enable_stonith:

except first where resource created

Reply:
Done.